src/stage1/ir.cpp

```diff
@@ -3338,6 +3338,32 @@ static void float_div_floor(ZigValue *out_val, ZigValue *op1, ZigValue *op2) {
     }
 }
 
+// c = a - b * trunc(a / b)
+static float16_t zig_f16_rem(float16_t a, float16_t b) {
```

Comment:
why is this called `rem`, if before it was called `mod` ?
[modulo != remainder](https://stackoverflow.com/a/13683709) for negative numbers.
Add test cases with negative numbers.

Reply:
If you look at rounding, `rem` and `mod` do them differently.

Reply:
ah, sounds good. mark this as resolved, if you want.